.mailmap

```diff
@@ -0,0 +1,50 @@
+Anibal Sólon Heinsfeld <anibalsolon@gmail.com>
+Christopher J. Markiewicz <effigies@gmail.com>
+Christopher J. Markiewicz <effigies@gmail.com> <effigies@bu.edu>
+Christopher J. Markiewicz <effigies@gmail.com> <markiewicz@stanford.edu>
+Daniel J. Lurie <danjlurie@gmail.com>
+Daniel J. Lurie <danjlurie@gmail.com> Daniel Lurie <danjlurie@gmail.com>
```

Comment:
The first line will map every instance with that email to that name. Because this line (and the following) have the same email, they're redundant.